src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -202,50 +201,54 @@ def __init__(self, target: Text,
 
         self._channel = cygrpc.AioChannel(_common.encode(target))
 
-    def check_connectivity_state(self, try_to_connect: bool = False
-                                ) -> grpc.ChannelConnectivity:
+    def get_state(self,
+                  try_to_connect: bool = False) -> grpc.ChannelConnectivity:
         """Check the connectivity state of a channel.
 
         This is an EXPERIMENTAL API.
 
+        It's the nature of connectivity states to change. The returned
+        connectivity state might become obsolete soon. Combining
+        "Channel.wait_for_state_change" we guarantee the convergence of
+        connectivity state between application and ground truth.
+
         Args:
-          try_to_connect: a bool indicate whether the Channel should try to connect to peer or not.
+          try_to_connect: a bool indicate whether the Channel should try to
+            connect to peer or not.
 
         Returns:
           A ChannelConnectivity object.
         """
         result = self._channel.check_connectivity_state(try_to_connect)
-        return _common.CYGRPC_CONNECTIVITY_STATE_TO_CHANNEL_CONNECTIVITY[result]
+        return _common.CYGRPC_CONNECTIVITY_STATE_TO_CHANNEL_CONNECTIVITY.get(
+            result)
 
-    async def watch_connectivity_state(
+    async def wait_for_state_change(
             self,
             last_observed_state: grpc.ChannelConnectivity,
-            timeout_seconds: Optional[float] = None,
-    ) -> Optional[grpc.ChannelConnectivity]:
-        """Watch for a change in connectivity state.
+    ) -> None:
+        """Wait for a change in connectivity state.
 
         This is an EXPERIMENTAL API.
 
-        Once the channel connectivity state is different from
-        last_observed_state, the function will return the new connectivity
-        state. If deadline expires BEFORE the state is changed, None will be
-        returned.
+        The function blocks until there is a change in the channel connectivity
+        state from the "last_observed_state". If the state is already
+        different, this function will return immediately.
 
-        Args:
-          try_to_connect: a bool indicate whether the Channel should try to connect to peer or not.
+        There is an inherent race between the invocation of
+        "Channel.wait_for_state_change" and "Channel.get_state". The state can
```

Comment:
I think this sentence is missing a verb. "change"?

Reply:
You are right. Fixed.